CONTRIBUTING.md

```diff
@@ -566,56 +566,60 @@ Names of files and directories should be in lowercase, with dashes between words
   ```nix
   foo
   {
-    arg = ...;
+    arg = <...>;
```

Comment:
Possibly the Technically Correct way of doing this would be using guillemets, since angle brackets are lookup path syntax. `nix-instantiate` uses those too for special values, so arguably that would increase consistency.